src/Main/ProgramSettings.cc

```diff
@@ -197,6 +197,7 @@ void ProgramSettings::ApplyCommandLineSettings(int argc, char** argv) {
     options.add_options("Deprecated and debug")
         ("debug_no_auth", "accept all incoming WebSocket connections on the specified port(s) (not secure; use with caution!)", cxxopts::value<bool>())
         ("no_runtime_config", "do not send a runtime config object to frontend clients", cxxopts::value<bool>())
+        ("controller_build", "used when the backend is hosted for carta-controller", cxxopts::value<bool>())
```

Comment:
This should be renamed to `controller_deployment` -- this is not a kind of backend build.

`hosted for` should be `launched by`.